acquia_lift_profiles/acquia_lift_profiles.install

```diff
@@ -180,3 +180,19 @@ function acquia_lift_profiles_update_7002(&$sandbox) {
     return t('All actions were synchronized to Lift Web');
   }
 }
+
+/**
+ * Convert existing vocabulary mappings to use taxonomy context module.
+ */
+function acquia_lift_profiles_update_7003() {
+  $vocabulary_mappings = variable_get('acquia_lift_profiles_vocabulary_mappings');
+  // Migration: prefix to existing mappings (if not already).
+  // Before: array ('', '7', '82', 'taxonomy_context__4')
+  // After: array ('', 'taxonomy_context__7', 'taxonomy_context__82', 'taxonomy_context__4')
+  $vocabulary_mappings_updated = preg_replace('/^(\d+)$/', 'taxonomy_context' . preg_quote(PERSONALIZE_TARGETING_ADMIN_SEPARATOR) . '$1', $vocabulary_mappings);
+  // If taxonomy is already configured, enable the taxonomy context module.
+  if ($vocabulary_mappings != $vocabulary_mappings_updated) {
+    module_enable(array('personalize_taxonomy_context'));
+  }
+  variable_set('acquia_lift_profiles_vocabulary_mappings', $vocabulary_mappings_updated);
+}
```

Comment:
@katbailey You at one point noted "you'd also need to make acquia_lift_profiles module declare a dependency on personalize_taxonomy". I now think this may actually not necessarily the case. Pfizer/Tim's use case may just be his custom context module, and no taxonomy. Because using taxonomy as the vocabulary mapping is voluntary, I did not declare acquia_lift_profiles' dependencies on taxonomy_context and taxonomy, and left the choice open to the user.

I understand that I may be mistaken the use cases here, so please let me know haha.

Two things at migration:
1) Update the saved value by prefixing the vocabulary id.
2) Only enable the personalize_taxonomy_context if it is already in use before this update hook.